templates/views/events.html

```diff
@@ -78,15 +78,15 @@ <h4 class="event_topic4">React Ladies Bootcamp</h4>
                 <h4 class="event_topic3">Code Literacy Day</h4>
                 <p class="event_topic2">Participate in any of our online or offline events and trainings: Technical and
                     non-Technical Events, Meetups, Mentoring Program</p>
-                <button class="btn event_button">SEE EVENT GALLERY</button>
+                <a class="btn event_button" a href="/events/event">SEE EVENT GALLERY</a>
             </div>
         </div>
         <div class="row row_padding">
             <div class="col-md-7 col-lg-7 col-sm-7 topic_padding">
                 <h4 class="event_topic4">Summer Break Project</h4>
                 <p class="event_topic2">Sponsor our next event via a donation or Partner with our organisation to reach
                     more women in tech across Africa and help us make an impact! Your support would go a long way.</p>
-                <button class="btn event_button">SEE EVENT GALLERY</button>
+                <a class="btn event_button" a href="/events/event">SEE EVENT GALLERY</a>
```

Comment:
Instead of the URL being `/events/event`, is it possible to have the name of the event instead? 

For example: `events/code-literacy-day`

Reply:
Yes, it is. It is like that for now because the back end hasn't been implemented on this branch yet. That will cater for linking to unique event gallery.